doc/schemas/signinvoice.request.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "invstring"
```

Comment:
This is not related to this PR but maybe we should write down some naming convention about the string invoice to have the same key around different API 

See https://lightning.readthedocs.io/lightning-decode.7.html